tables/variables.txt

```diff
@@ -18002,9 +18002,9 @@
     0x80BE61A4:("enTanron6OverlayRelocations","u32","[9]",0x24),
     0x80BE61CC:("enTanron6OverlayInfoOffset","u32","",0x4),
     0x80BE78D0:("En_Daiku2_InitVars","UNK_TYPE1","",0x1),
-    0x80BE78F0:("D_80BE78F0","UNK_TYPE1","",0x1),
+    0x80BE78F0:("sTextIDs","UNK_TYPE1","",0x1),
```

Comment:
```suggestion
    0x80BE78F0:("sTextIds","UNK_TYPE1","",0x1),
```
The convention I've seen is to have `Id` instead of `ID`, could you change the uses of this throughout the files?